app/src/app/ui/dynamic-theme/custom-background-color.scss

```diff
@@ -0,0 +1,16 @@
+@function get-custom-backgrond-color($background-color, $theme) {
+    $background: map-get($theme, background);
+
+    $card: map-get($background, card);
+
+    $new-card-color: lighten($background-color, 10%);
+
+    $new-selected-button: rgba(255, 255, 255, 20%);
```

Comment:
not sure about this, in adf I see this         --theme-selected-button-bg-color: mat-color($background, selected-button),

Reply:
in material design selected button is darker variation of background colour (both colours comes from the same palette):

```
background: map-get($mat-grey, 50),
selected-button: map-get($mat-grey, 300),
```
Therefore I added white with transparent in order to make the colour more darker. 

But yes, if someone will use it outside of background colour, we will get different colour.

Changed to:

```
$new-card-color: lighten($background-color, 10%);
$new-selected-button: lighten($background-color, 5%);
```